sdk/core/ts-http-runtime/src/util/sanitizer.ts

```diff
@@ -162,7 +162,7 @@ export class Sanitizer {
     return sanitized;
   }
 
-  private sanitizeUrl(value: string): string {
+  public sanitizeUrl(value: string): string {
```

Comment:
Passing thought, I wonder if this really needs to be a class; it barely has state. Of course there are perf implications and we wouldn't want to construct all the sets every request but I wonder if there's a way we could avoid doing that?

Anyways, not in scope for this PR, so feel free to ignore this comment :)

Reply:
Agree 100% - we'd still need to manage the set of allowed values and those are configurable at the client constructor level but there are probably better ways to do that.

I will not make that change here, just to keep changes minimal but I deff agree.